arviz/plots/ppplot.py

```diff
@@ -0,0 +1,181 @@
+"""Posterior-Prior plot."""
+
+from .plot_utils import (
+    xarray_var_iter,
+    _scale_fig_size,
+    get_plotting_function,
+)
+from ..utils import _var_names, get_coords
+from ..rcparams import rcParams
+
+
+def plot_pp(
+    data,
+    kind="latent",
+    figsize=None,
+    textsize=None,
+    var_names=None,
+    coords=None,
+    transform=None,
+    legend=True,
+    ax=None,
+    prior_kwargs=None,
+    posterior_kwargs=None,
+    observed_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    show=None,
+):
+    """
+    Plot for posterior/Prior.
```

Comment:
Maybe "Plots that compare fitted and unfitted distributions." as the one-liner?

Then maybe a more extended description of exactly what the comparisons will be?

E.g.

"The resulting plots will show the compared distributions (as KDEs) both on separate axes (particularly useful when one of them is substantially tighter than another), and plotted together, so three plots per distribution.

The groups plotted are dictated by the value of the ``kind`` argument."